src/frontend/src/icp/components/send/IcSendForm.svelte

```diff
@@ -19,12 +19,15 @@
 	export let amount: OptionAmount = undefined;
 	export let networkId: NetworkId | undefined = undefined;
 	export let source: string;
+	export let simplifiedForm = false;
 
 	const { sendToken } = getContext<SendContext>(SEND_CONTEXT_KEY);
 
 	let amountError: IcAmountAssertionError | undefined;
 	let invalidDestination: boolean;
 
+	let sourceValue = simplifiedForm ? undefined : source;
```

Comment:
I missed this in my previous review. I might be wrong, but forcing a value to undefined to avoid displaying it seems like an anti-pattern with respect to OISY's codebase. Additionally, it appears the field still accepts undefined but is never actually undefined. Therefore, I believe it would be better to express the choice of not displaying the source by using a dedicated property.